waddrmgs/manager.go

```diff
@@ -51,6 +51,9 @@ const (
 	AccountMergePayNum=-1
 
 
+	Default_send_minconf =1
```

Comment:
The definition is not good enough than before, It's still hard for the code reader to understand the meaning of the `SendPairs ` function calling convention.